evremap.service

```
@@ -3,7 +3,7 @@ WorkingDirectory=/
 # For reasons I don't care to troubleshoot, Fedora 31 won't let me start this
 # unless I use `bash -c` around it.  Putting the command line in directly
 # yields a 203 permission denied error with no logs about what it didn't like.
-ExecStart=bash -c "/home/wez/github/evremap/target/release/evremap remap /home/wez/github/evremap/pixelbookgo.toml"
+ExecStart=bash -c "/usr/bin/evremap remap /etc/evremap/remap.toml"
```

I'm not sure that we need a new directory for a simple single file config, so I would lean towards this:

```suggestion
ExecStart=bash -c "/usr/bin/evremap remap /etc/evremap.toml"
```

---

I also ran into an issue with the `WantedBy` target (no GDM). Are you OK with changing it to a generic target like `multi-user.target` or similar?

---

Yeah, go for it!